WordPress/src/main/java/org/wordpress/android/ui/pages/PagesFragment.kt

```diff
@@ -264,6 +278,26 @@ class PagesFragment : Fragment() {
             savedInstanceState.getSerializable(WordPress.SITE) as SiteModel
         }
 
+        viewModel.authorUIState.observe(activity, Observer { state ->
+            state?.let {
+                val authorSelectionVisibility = if (state.isAuthorFilterVisible) View.VISIBLE else View.GONE
```

Comment:
We could use `UiHelpers.updateVisibility()` here. But it's not a big deal.

Reply:
TIL - thanks. Changed in the next commit.